src/m365/spfx/commands/spfx-doctor.ts

```diff
@@ -651,189 +655,290 @@ class SpfxDoctorCommand extends BaseProjectCommand {
       args.options.output = 'text';
     }
 
+    this.output = args.options.output;
     this.projectRootPath = this.getProjectRoot(process.cwd());
+    this.logger = logger;
+
+    await this.logMessage(' ');
```

Comment:
why do we add this twice? here and also in line 664?

Reply:
Think it was a merge thingy. My bad